beacon_node/beacon_chain/src/eth1_chain.rs

```diff
@@ -98,7 +98,7 @@ fn get_sync_status<T: EthSpec>(
     } else {
         // The number of seconds in an eth1 voting period.
         let voting_period_duration =
-            T::slots_per_eth1_voting_period() as u64 * (spec.milliseconds_per_slot / 1_000);
+            T::slots_per_eth1_voting_period() as u64 * spec.milliseconds_per_slot;
```

Comment:
wait a sec, what's going on here?

bors r-

Reply:
Ok, I see. The comment above should _definitely_ be updated though, and maybe the name of the variable. `voting_period_duration_millis`?

Reply:
Or do we even need to change this if we only support seconds anyway?

Reply:
Sorry @blacktemplar, just me being paranoid :sweat_smile:

Reply:
Oh yes, this was me trying to fix slot durations that are not multiple of seconds before I recognized the more fundamental problem of having only `seconds_per_slot` in `YamlConfig`. https://github.com/sigp/lighthouse/pull/2083 superseeds those changes anyway, so I am happy to revert them :).